packages/components/src/item-group/styles.ts

```diff
@@ -42,6 +42,7 @@ export const unstyledButton = ( as: 'a' | 'button' ) => {
 			// Windows high contrast mode.
 			outline: 2px solid transparent;
 			outline-offset: 0;
+			border-radius: 2px;
```

Comment:
This is a global change to the component. I think styling needs to be block-type item list specific.